Changes.md

```diff
@@ -1,6 +1,11 @@
 0.57.x.x (relative to 0.57.1.0)
 ========
 
+Fixes
+-----
+
+- IECoreArnoldPreview::Renderer : Hack around Arnold failure to update quad lights
```

Comment:
This is the user-facing change log used for release announcements etc. In that context `IECoreArnoldPreview` is irrelevant and "hack around" isn't suitable language.

```suggestion
- Arnold : Added workaround for Arnold bug which prevented interactive edits to quad light colour textures.
```